### DISCUSSION TITLE
Common theme for our viz gallery

### DISCUSSION THREAD

**Discussion by michael-s-molina** [apache/superset#29076] [Ideas]
As suggested by @john-bodley in https://github.com/apache/superset/pull/28789#issuecomment-2142385107, we could apply a common theme to our viz gallery thumbnails and examples to uplevel the visual quality.

@kasiazjc @rusackas @yousoph

**Comment by rusackas** [comment]
Hmm... maybe. They would all look conformant if we do so, but honestly I think the fact that you can run wild with color schemes (political maps, hot/cold, brand palettes, etc) is one of the strengths of Superset. It might make the product look _less_ capable than it is. Not all of these have meaning/utility, but many do. I would not want to see a red/yellow/green andon chart, or a red/green product growth chart, turned into some arbitrary colors just to be pretty/thematic.

In doing so, you'll also likely run into some of the more problematic color palette issues. For example, assigning a categorical palette to a dashboard wrecks charts that benefit from a gradient palette. Dashboards don't allow you to provide both (yet).
